api/v6/report_server.thrift

```diff
@@ -94,7 +110,10 @@ struct ReportFilter_v2 {
   4: list<string>                 reportHash,
   5: list<shared.Severity>        severity,
   6: list<shared.ReviewStatus>    reviewStatus,
-  7: list<shared.DetectionStatus> detectionStatus
+  7: list<shared.DetectionStatus> detectionStatus,
+  8: list<string>                 runHistoryTag,
+  9: optional string              detectedDate,
```

Comment:
I think `detectionDate` or `firstDetectionDate` would be more expressive. A comment would be nice that this value is set only once.